mappings/net/minecraft/entity/passive/FrogBrain.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS net/minecraft/unmapped/C_gtwuaauj net/minecraft/entity/passive/FrogBrain
+	FIELD f_bguhkofh MAKING_LOVE_SPEED F
+	FIELD f_fbhvaoaw MAX_LONG_JUMP_HEIGHT I
+	FIELD f_hstykbvl WATER_SPEED F
+	FIELD f_hxbjscly MAX_JUMP_VELOCITY F
+	FIELD f_nikiifks IDLING_SPEED F
+	FIELD f_ookmoifl LONG_JUMP_COOLDOWN_RANGE Lnet/minecraft/unmapped/C_hvilmtvi;
+	FIELD f_viidhanx TEMPTED_SPEED_MULTIPLER F
+	FIELD f_wlgvvjll PANICKING_SPEED_MULTIPLER F
+	FIELD f_xqntdhtx LAND_SPEED F
+	FIELD f_xrxbfjsi MAX_LONG_JUMP_DISTANCE I
+	METHOD m_andsotcf (Lnet/minecraft/unmapped/C_usxaxydn;)Ljava/lang/Float;
+		ARG 0 entity
+	METHOD m_czziutiw (Lnet/minecraft/unmapped/C_txtbiemp;)Z
+		ARG 0 state
+	METHOD m_euyxgnus (Lnet/minecraft/unmapped/C_gcfircge;)Ljava/util/Optional;
+		ARG 0 frog
+	METHOD m_fhaglmeu addIdleActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_fmdizgqd (Lnet/minecraft/unmapped/C_gcfircge;)Lnet/minecraft/unmapped/C_avavozay;
+		ARG 0 entity
+	METHOD m_ihwohwid addLongJumpActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_ioifnvht canAttack (Lnet/minecraft/unmapped/C_gcfircge;)Z
+		ARG 0 frog
+	METHOD m_iyhjlwzh (Lnet/minecraft/unmapped/C_gcfircge;)Ljava/util/Optional;
+		ARG 0 frog
+	METHOD m_kbsketlv addCoreActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_kitbydka initializeMemories (Lnet/minecraft/unmapped/C_gcfircge;Lnet/minecraft/unmapped/C_rlomrsco;)V
+		ARG 0 frog
+		ARG 1 generator
+	METHOD m_lixpcywj reset (Lnet/minecraft/unmapped/C_gcfircge;)V
+		ARG 0 frog
+	METHOD m_lvidokhl create (Lnet/minecraft/unmapped/C_rjqjaxef;)Lnet/minecraft/unmapped/C_rjqjaxef;
+		ARG 0 brain
+	METHOD m_lxhweavq addSwimActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_mtqywzuf addLaySpawnActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_thifxbji (Lnet/minecraft/unmapped/C_usxaxydn;)Ljava/lang/Float;
+		ARG 0 entity
+	METHOD m_tunjbuuy addTongueActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_uzrluyop getTemptItems ()Lnet/minecraft/unmapped/C_tcpsydrv;
```

Comment:
What exactly are tempt items?

Reply:
Basically items the frog may like and will follow you based on it